### DISCUSSION TITLE
Welcome to geotiff Discussions!

### DISCUSSION THREAD

**Discussion by KipCrossing** [KipCrossing/geotiff#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by UfotOnoTofu** [comment]
Hello.  I work for a company that uses ESRI ArcGIS software for forestry purposes.  I'm a database developer and not much experience with what all the company is using with the software as far as visualization, etc.  I know they use it for marking prospective construction or maintenance sites in the field.   I'm using python and looking to build a layer viewer from whatever sources I can find. For predictive purposes.  Does ArcGIS already offer this kind of functionality?  Right now I'm just trying to read layer type from the geotiff file that has been downloaded.  I'm beginning to think that downloading is a bottleneck, and I should be reading data from web sources on the fly as I drill down.  Any and all thoughts appreciated on these points.
